test/e2e/infra/admin_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Project Contour Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build e2e
+
+package infra
+
+import (
+	. "github.com/onsi/ginkgo"
+	"github.com/projectcontour/contour/test/e2e"
+	"github.com/stretchr/testify/require"
+)
+
+func testAdminInterface() {
+	Specify("requests to admin listener are served", func() {
```

Comment:
maybe add a test to make sure the endpoints that can modify envoy server settings are not available/we get connection errors/404s etc.

Reply:
👍